src/controllers/users.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const userServices = require('../services/users');
+
+const getAll = async(req, res) => {
+  const users = await userServices.getAll();
```

Comment:
wrapp all async operations in try catch(or use then catch) to avoid uncatched errors